.github/workflows/ci.yml

```
@@ -0,0 +1,107 @@
+
+jobs:
+  test:
+    name: Test Suite
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        rust: [stable, beta, nightly]
+    steps:
+      - uses: actions/checkout@v4
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)